os/board/rtl8720e/src/component/bluetooth/api/rtk_bt_common.c

```diff
@@ -748,8 +748,8 @@ uint16_t rtk_bt_set_evt_cb_direct_calling(uint8_t group, uint32_t evt_bit_mask)
 
 	direct_call_t.group = group;
 	direct_call_t.evt_bit_mask = evt_bit_mask;
-	ret = bt_ipc_api_host_message_send(RTK_BT_IPC_COMMON, RTK_BT_ACT_IPC_ENABLE_DIRECT_CALL,
-										&direct_call_t, sizeof(rtk_bt_direct_call_t));
+	ret = bt_ipc_api_host_message_send(RTK_BT_IPC_COMMON, RTK_BT_ACT_IPC_ENABLE_DIRECT_CALL, 
+									(uint8_t*)&direct_call_t, sizeof(rtk_bt_direct_call_t));
```

Comment:
Add space between the type & asterisk for all the type casts

Reply:
I will make the changes again

Reply:
@kaikaikaime @thapav We don't recommend to follow TizenRT coding rules in the BSP (chipset vendor codes). @kaikaikaime You don't need to follow it here. You can decide and let us know about modification (will do or not).

Reply:
@kaikaikaime Could you let me know your plan? Will you keep current (merge current) or will make changes (wait your changes)?

Reply:
Hi @sunghan-chang, I may need to discuss with my colleague on this, I will get back to you latest at the end of the day.